inference-engine/thirdparty/clDNN/src/impls/onednn/pooling_onednn.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pooling_inst.h"
+#include "primitive_onednn_base.h"
+#include "impls/implementation_map.hpp"
+
+#include "kernel_selector_common.h"
+
+#include <oneapi/dnnl/dnnl.hpp>
+
+#include <algorithm>
+#include <memory>
+namespace cldnn {
+namespace onednn {
+
+
+namespace detail {
```

Comment:
```suggestion
namespace onednn {

namespace detail {
```

Reply:
I expect this file to be replaced by the actual file that other person work. So it is OK to merge though there are some ugliness :(